packages/forms/resources/js/components/markdown-editor.js

```diff
@@ -210,6 +211,10 @@ export default function markdownEditorFormComponent({
 
                 Alpine.raw(this.editor).value(this.state ?? '')
             })
+
+            if (afterInit) {
+                afterInit.apply(this)
```

Comment:
Can afterInit be a simple function instead of an object with a functional property?

Reply:
I don't understand what you mean. Could you give me an example of what you're thinking?

Reply:
```suggestion
                afterInit(this)
```

Reply:
ok, so the function would receive an instance of the markdown editor component object, instead of having to resolve it via `this`? I'm ok with that.

Reply:
oh wait, .apply() is some feature of JS to set `this`? I didn't know